dll/win32/browseui/addressband.cpp

```diff
@@ -259,24 +260,53 @@ HRESULT STDMETHODCALLTYPE CAddressBand::HasFocusIO()
     return S_FALSE;
 }
 
+WCHAR CAddressBand::GetFocusKey()
+{
+    if (m_chFocusKey != UNICODE_NULL)
+        return m_chFocusKey;
+
+    CStringW strLabel(MAKEINTRESOURCEW(IDS_ADDRESSBANDLABEL));
+    INT ich = strLabel.Find(L'&');
+    if (ich == -1)
+        m_chFocusKey = L'D'; /* Alt+D */
+    else
+        m_chFocusKey = (WCHAR)(INT_PTR)CharUpperW((LPWSTR)(INT_PTR)strLabel[ich + 1]);
+
+    return m_chFocusKey;
```

Comment:
Ah yes, whatever way we do, because the key is cached and not re-updated everytime, when one changes the UI language of ReactOS at runtime (with an explorer window opened), say, from english to german, the UI would change (and the "Address" label would change to the german corresponding one), but the cached accelerator would still be (Alt+)D instead of another one. This is a bug.

Reply:
I tried to fix this UI bug by using newly-added `m_strAddressLabel`.